src/Label/ReadModels/JSON/Repository/Doctrine/DBALReadRepository.php

```diff
@@ -148,7 +148,7 @@ private function createSearchQuery(Query $query): QueryBuilder
             );
 
         if ($query->isSuggestion()) {
-            $queryBuilder->andWhere('name REGEXP \'^[a-zA-Z\d_\-]{2,50}$\'');
```

Comment:
Our version of `sqlite` doesn't support `REGEX` so this cannot be unit tested.

Reply:
Put the `REGEX` inside the `LabelName` class and rename the existing one to `LEGACY_REGEX`.
Just use `preg_match` on the public `REGEX` for intensive testing.